### PR TITLE
Add team PPPs

### PR DESCRIPTION
**Pull request by zhyuhan**

### CODE REVIEW

**Review by KamiliArsyad — COMMENTED** [review]
LGTM

**Review by KamiliArsyad — APPROVED** [review]

File: docs/team/dickongwd.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: dickongwd's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```
Comment (dickongwd) [inline docs/team/dickongwd.md]: Could you change AddressBook Level 3 to Connectify?

File: docs/team/kamiliarsyad.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: KamiliArsyad's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```
Comment (dickongwd) [inline docs/team/kamiliarsyad.md]: Our product name is Connectify

File: docs/team/lilozz2.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: lilozz2's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```
Comment (dickongwd) [inline docs/team/lilozz2.md]: Our product name is Connectify

File: docs/team/magnificentcreature.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: MagnificentCreature's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```
Comment (dickongwd) [inline docs/team/magnificentcreature.md]: Our product name is Connectify

File: docs/team/zhyuhan.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: zhyuhan's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```
Comment (dickongwd) [inline docs/team/zhyuhan.md]: Our product name is Connectify

**Review by dickongwd — CHANGES_REQUESTED** [review]
Just minor changes for the product name!

**Comment by zhyuhan** [comment]
@dickongwd Good catch, thanks!

**Comment by dickongwd** [comment]
LGTM!

**Review by dickongwd — APPROVED** [review]
LGTM!

**Pull request merged**